lib/dnsimple/client.rb

```diff
@@ -173,8 +173,8 @@ def base_options
       options = {
           format:   :json,
           headers:  {
-            'Accept' => 'application/json',
-            'User-Agent' => user_agent
+              'Accept' => 'application/json',
```

Comment:
Not sure how I feel about this indentation. I think I'd rather stick with the standard 2, but if it's automatically cleaned up, I guess I don't care that much.